algorithm/cgal/main.js

```diff
@@ -70,6 +77,7 @@ export { generatePackingEnvelopeForSurfaceMesh } from './generatePackingEnvelope
 export { grow } from './grow.js';
 export { initCgal } from './getCgal.js';
 export { inset } from './inset.js';
+export { involute } from './involute.js';
```

Comment:
Are these all exposed in the api? a.involute, a.mirror...etc

Reply:
involute is, mirror needs deleting. :)